gcc/d/ChangeLog

```diff
@@ -1,3 +1,7 @@
+2017-10-15  Eugene Wissner <belka@caraus.de>
+
+	* patches/patch-targetdm-8.x: Update gcc/config.gcc diff.
```

Comment:
Don't need a changelog entry for patches. Only for in-tree sources.

Reply:
I'll note it.